applications/crossbar/src/modules/cb_rates.erl

```diff
@@ -53,18 +53,25 @@ init() ->
     _ = crossbar_bindings:bind(<<"*.execute.patch.rates">>, ?MODULE, 'patch'),
     crossbar_bindings:bind(<<"*.execute.delete.rates">>, ?MODULE, 'delete').
 
+-spec init_db() -> 'ok'.
 init_db() ->
-    _ = couch_mgr:db_create(?WH_RATES_DB),
-    couch_mgr:revise_doc_from_file(?WH_RATES_DB, 'crossbar', "views/rates.json").
+    init_db(?WH_RATES_DB).
 
+-spec authorize(cb_context:context()) -> boolean().
 authorize(Context) ->
-    authorize(Context, cb_context:req_nouns(Context)).
+    AccId = cb_context:account_id(Context),
```

Comment:
Please use `AccountId` instead; we're trying to avoid abbreviations, especially when it could conflict with other contexts (in this case `Acc` is usually short for `Accumulator` in a fold).